internal/profile/profile.go

```diff
@@ -1,10 +1,10 @@
-package cli
+package profile
```

Comment:
This felt a little dirty but I had to move profile out of cli due to dependency cycles. It is a bit weird because of how much profile is a part of the components in the cli package but I couldn't find a better way for the client to have save the new auth token to the profile